Taskfile.yml

```diff
@@ -257,7 +264,7 @@ tasks:
         bin/ferretdb-testcover{{exeExt}} -test.coverprofile=cover.txt --
         --listen-addr=:27017
         --proxy-addr=127.0.0.1:47017
-        --mode=diff-normal
+        --mode=normal
```

Comment:
Is this change needed because of the replica set?

Reply:
Actually that's a mistake. I will revert that change.